tasks/config/licenses.js

```diff
@@ -32,6 +32,7 @@ module.exports = function (grunt) {
         'UNLICENSE'
       ],
       overrides: {
+        'abbrev@1.0.7': ['MIT'],
```

Comment:
I'm not familiar with how we decide whether its okay to add a package to this list. Perhaps @spalger or @epixa might know?